README.md

```diff
@@ -1,55 +1,111 @@
+
+</div>
+
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-- [💻 Getting Started](#getting-started)
-- [Key feature](#Key feature)
- -[🚀 Live Demo](#live-demo)
-- [Setup](#setup)
-- [Run tests](#run-tests)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [📝 License](#license)
-
-# JavaScript-Capstone-Group-Project-TV-API
-The JavaScript capstone project is about building our web application based on an external API. We've selected a TV-based API and then build our web app around it. The web app has 2 user interfaces.
-
-## Built With
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 LEADERBOARD ](#-Leaderboard-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magicians<a name="about-project"></a>
+
+Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+-[] Make simple calculations.
+-[] Read a random math-related quote.
+This application help you to learn how to use React.
+
+## 🛠 Built With <a name="built-with"></a>
 HTML
 CSS
 JavaScript (ES6)
 Linters
 React
 
-## Key feature
-Here are some key features of the application:
-Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+### Tech Stack <a name="tech-stack"></a>
 
--[] Make simple calculations.
--[] Read a random math-related quote.
+> This project uses the following stack :
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+     <li><a href="https://www.w3schools.com/js/">JavaScript</a></li>
+     <li><a href="hhttps://reactjs.org/">ReactJs</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Here are some key features of the application :
+
+- **Learn how to use proper ES6 syntax.**
+- **Use ES6 modules to write modular JavaScript.**
+- **Follow Gitflow.**
+- **Handle events in a React app.**
+- **Use React life cycle methods.**
+- **Understand the mechanism of lifting state up.**
 
-## Important Links
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](Coming soon)
+[Math Magicians](coming soon))
```

Comment:
- [ ] Kindly either remove this or add a proper link here to make your readme professional.

Reply:
Thanks for the review